srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -30,10 +30,11 @@ global
     maxconn     {{ (150 / salt['pillar.get']('haproxy:nbproc', 2)) | round(0, 'ceil') | int }}
     user        haproxy
     group       haproxy
+    nbproc	1
     {%- if "physical" in grains['virtual'] %}
-    nbproc      {{ salt['pillar.get']('haproxy:nbproc', 2) }}
+    nbthread     16
```

Comment:
Please don't hardcode the values, it needs to be captured somewhere in the pillar configuration  (e.g. haproxy:nbthread) and then set it here.